src/components/Navigator/NavigatorDataProvider.vue

```diff
@@ -45,6 +47,18 @@ export default {
     };
   },
   computed: {
+    /**
+     * Recomputes the list of flat children.
+     * @return NavigatorFlatItem[]
+     */
+    flatChildren: ({
+      flattenNestedData, technologyWithChildren = {},
+    }) => {
+      const flatIndex = flattenNestedData(
```

Comment:
No need for a variable here, just return it.

Reply:
Resolved [c17ff68](https://github.com/apple/swift-docc-render/pull/396/commits/c17ff680b73bf74237d8cf89054fcb7cd651b07a)